lib/async/future_async.ml

```diff
@@ -68,7 +68,7 @@ module Unix = struct
   (* let getpid = Unix.getpid *)
 
   module Stats = struct
-    type t = Core.Unix.stats = {
+    type _t = Core.Unix.stats = {
```

Comment:
Could you explain this change?

Reply:
@pveber This type was unused, causing a compiler warning to trigger (which now would break the build).